test/1_stdlib/UnfoldSequence.swift

```diff
@@ -0,0 +1,48 @@
+// RUN: %target-run-simple-swift
```

Comment:
We could, but validation tests are run much less often so I kind of assumed anything that is reasonable to put into the normal test suite, should be put into the normal test suite. Am I wrong?